app/src/main/java/com/example/concordia_campus_guide/Activities/SearchActivity.java

```diff
@@ -28,4 +100,12 @@ public void onClick(View v) {
             }
         });
     }
+
+    private String getPlaceType(Place place){
+        if(place instanceof RoomModel) return RoomModel.class.getSimpleName();
+        if(place instanceof Floor) return Floor.class.getSimpleName();
+        if(place instanceof Building) return Building.class.getSimpleName();
+
+        return null;
+    }
```

Comment:
Do you think we can place this method in the viewModel ?

Reply:
deleted the method